flojoy/utils.py

```diff
@@ -45,30 +47,9 @@
 
 
 # Make as a function to mock at test-time
-def _get_hf_hub_cache_path() -> str:
-    return os.path.join(FLOJOY_CACHE_DIR, "cache", "hf_hub")
-
-
-def hf_hub_download(*args, **kwargs):
-    if "cache_dir" not in kwargs:
-        kwargs["cache_dir"] = _get_hf_hub_cache_path()
-    else:
-        if kwargs["cache_dir"] != _get_hf_hub_cache_path():
-            raise ValueError(
-                f"Attempted to override cache_dir parameter, received {kwargs['cache_dir']} while the only alloed value is {_get_hf_hub_cache_path()}"
-            )
-    return _hf_hub_download(*args, **kwargs)
-
-
-def snapshot_download(*args, **kwargs):
-    if "cache_dir" not in kwargs:
-        kwargs["cache_dir"] = _get_hf_hub_cache_path()
-    else:
-        if kwargs["cache_dir"] != _get_hf_hub_cache_path():
-            raise ValueError(
-                f"Attempted to override cache_dir parameter, received {kwargs['cache_dir']} while the only alloed value is {_get_hf_hub_cache_path()}"
-            )
-    return _snapshot_download(*args, **kwargs)
+def get_hf_hub_cache_path() -> str:
+    """Returns the path to the HuggingFace cache directory within the Flojoy cache directory"""
+    return os.path.join(FLOJOY_CACHE_DIR, "cache", "hf_home")
```

Comment:
For consistency with `~/.cache/huggingface`, [the default definition of `HF_HOME`](https://huggingface.co/docs/huggingface_hub/package_reference/environment_variables#hfhome).

```suggestion
    return os.path.join(FLOJOY_CACHE_DIR, "cache", "huggingface")
```

Also, note that `HF_HOME` is the global folder for Hugging Face Hub, and not just for its difference cache. [`HUGGINGFACE_HUB_CACHE`](https://huggingface.co/docs/huggingface_hub/package_reference/environment_variables#huggingfacehubcache) and [`HUGGING_FACE_HUB_TOKEN`](https://huggingface.co/docs/huggingface_hub/package_reference/environment_variables#huggingfacehubcache) do, have by default depends on `HF_HOME`.

I think we word the change of those PR for something like "Hugging Face Home" folder.

What do you think?

Reply:
Agreed, Home is more descriptive that "cache"